src/TestSuite/IntegrationTestCase.php

```diff
@@ -219,6 +229,43 @@ public function cookie($name, $value)
     }
 
     /**
+     * Returns the encryption key to be used.
+     *
+     * @return string
+     */
+    protected function _getCookieEncryptionKey() {
+        if (isset($this->_cookieEncriptionKey)) {
+            return $this->_cookieEncriptionKey;
+        }
+        $key = Security::salt();
+        if (isset($key)) {
+            return $key;
+        }
+        $key = Security::salt('foo!foo!foo!foo!foo!foo!foo!foo!');
```

Comment:
In some ways an exception is more useful as the code will not function outside of the test suite. Providing a default is easier for the user, but could mask configuration issues.